web/packages/design/src/Icon/Icons.story.tsx

```diff
@@ -16,54 +16,57 @@ limitations under the License.
 
 import React from 'react';
 
-import * as Icon from '../Icon';
+import { Text } from '..';
 
 import Flex from './../Flex';
 
+import * as Icon from '.';
+
 export default {
   title: 'Design/Icon',
 };
 
-export const DeprecatedIcons = () => (
+/*
+
+THIS FILE IS GENERATED. DO NOT EDIT.
+
+*/
+
+export const Icons = () => (
```

Comment:
```suggestion
export const PhosphorIcons = () => (
```

b/c we have a story called "PreferSvgIcon". We should change that to mean like... non phosphor icons?

Reply:
I think we should keep this as just `Icons` and rename the `PreferSvgIcon` story to something else, maybe `CustomIcons`?

Reply:
yeah lets do that 👍